CMakeLists.txt

```diff
@@ -266,6 +266,7 @@ if(BUILD_TESTS)
         set(BUILD_GMOCK OFF CACHE BOOL "Builds the googlemock subproject")
         set(gtest_force_shared_crt ON CACHE BOOL "Link gtest runtimes dynamically" FORCE)
         set(BUILD_SHARED_LIBS ON CACHE BOOL "Build shared libraries")
+        set(INSTALL_GTEST OFF CACHE BOOL "Don't install gtest")
```

Comment:
Currently installing with testing enabled produces the following undesirable output.

```
Run cmake --install build --prefix build/install
-- Install configuration: "Debug"
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/pkgconfig/vulkan.pc
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/libvulkan.so.1.3.250
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/libvulkan.so.1
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/libvulkan.so
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/cmake/GTest/GTestTargets.cmake
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/cmake/GTest/GTestTargets-debug.cmake
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/cmake/GTest/GTestConfigVersion.cmake
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/cmake/GTest/GTestConfig.cmake
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest_pred_impl.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/gtest-internal.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/gtest-string.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/gtest-type-util.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/gtest-death-test-internal.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/gtest-param-util.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/gtest-port-arch.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/gtest-filepath.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/gtest-port.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/custom
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/custom/README.md
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/custom/gtest.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/custom/gtest-port.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/internal/custom/gtest-printers.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest-param-test.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest-matchers.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest-death-test.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest-message.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest-spi.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest-typed-test.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest_prod.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest-printers.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/include/gtest/gtest-test-part.h
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/libgtestd.so.1.11.0
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/libgtestd.so
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/libgtest_maind.so.1.11.0
-- Set runtime path of "build/install/lib/libgtest_maind.so.1.11.0" to ""
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/libgtest_maind.so
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/pkgconfig/gtest.pc
-- Installing: /home/runner/work/Vulkan-Loader/Vulkan-Loader/build/install/lib/pkgconfig/gtest_main.pc
```